mlflow/types/schema.py

```diff
@@ -59,10 +61,27 @@ def to_spark(self):
 
         return getattr(pyspark.sql.types, self._spark_type)()
 
+    def to_python(self):
+        """Get equivalent python data type."""
+        return self._python_type
+
+    def get_all_types(self):
+        return [self.to_numpy(), self.to_pandas(), self.to_spark(), self.to_python()]
+
     @classmethod
     def get_spark_types(cls):
         return [dt.to_spark() for dt in cls._member_map_.values()]
 
+    @classmethod
+    # TODO: ADD TEST CASES
+    def is_instance(cls, value, data_type):
+        """
+        Check if the value is an instance of the given data type.
+        """
+        if data_type.name == "datetime":
+            return isinstance(value, np.datetime64)
+        return type(value) in data_type.get_all_types()
```

Comment:
`heck if the value is an instance of the given data type.` sounds like just do `isintance(value, data_type)` and can lead to confusion.

Reply:
/ai suggest names for this method.

Reply:
@mlflow Thanks! Let me know if you find any better names.